src/backend/metal/src/command.rs

```diff
@@ -137,6 +138,9 @@ impl CommandBufferInner {
         if let Some(ref pipeline_state) = self.pipeline_state {
             encoder.set_render_pipeline_state(pipeline_state);
         }
+        if let Some(ref depth_stencil_state) = self.depth_stencil_state {
```

Comment:
we also need to set the state if the encoder is already active in `bind_graphics_pipeline`

Reply:
Done